.github/workflows/gh-ci-tests.yaml

```
@@ -0,0 +1,58 @@
+name: mda_gh_ci_tests
+on:
+  push:
+    branches:
+      - develop
+  pull_request:
+    branches:
+      - develop
+  schedule:
+    # 3 am Tuesdays and Fridays
+    - cron: "0 3 * * 2,5"
+
+concurrency:
+  # Probably overly cautious group naming.
+  # Commits to develop/master will cancel each other, but PRs will only cancel
+  # commits within the same PR
+  group: "${{ github.ref }}-${{ github.head_ref }}-${{ github.workflow }}"
+  cancel-in-progress: true
+
+defaults:
+  run:
+    shell: bash -l {0}
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    name: "examples-tests"
+    strategy:
+      fail-fast: False
+      matrix:
+        os: ['ubuntu']
+        python-version: [3.8]
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: conda-incubator/setup-miniconda@v2
+        with:
+          auto-update-conda: True
+          use-mamba: True
+          python-version: ${{ matrix.python-version }}
+          miniforge-variant: Mambaforge
+          environment-file: environment.yml
+          activate-environment: mda-user-guide
+
+      - name: "environment information"
+        run: |
+          mamba info -a
+          mamba list
+      - name: "test notebooks"
```

```suggestion

      - name: "test notebooks"
```
Empty line for consistent formatting.

---

Fixed.